features/raadpleeg-met-reisdocumentnummer/gba/in-onderzoek-gba.feature

```diff
@@ -0,0 +1,71 @@
+    Als gba reisdocumenten wordt gezocht met de volgende parameters
+    | naam               | waarde                                        |
+    | type               | RaadpleegMetReisdocumentnummer                |
+    | reisdocumentnummer | NE3663258                                     |
+    | fields             | reisdocumentnummer,soort,datumEindeGeldigheid |
+    Dan heeft de response een 'reisdocument' met de volgende gegevens
+    | naam                                      | waarde                    |
+    | reisdocumentnummer                        | NE3663258                 |
+    | soort.code                                | PN                        |
+    | soort.omschrijving                        | Nationaal paspoort        |
+    | datumEindeGeldigheid                      | 20240506                  |
+    | inOnderzoek.aanduidingGegevensInOnderzoek | <aanduiding in onderzoek> |
+    | inOnderzoek.datumIngangOnderzoek          | 20230506                  |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | type                                                    |
+    | 120000                  | hele categorie reisdocument                             |
+    | 123500                  | hele groep Nederlands reisdocument                      |
+    | 123510                  | soort Nederlands reisdocument                           |
+    | 123520                  | nummer Nederlands reisdocument                          |
+    | 123530                  | datum uitgifte Nederlands reisdocument                  |
+    | 123540                  | autoriteit van uitgifte Nederlands reisdocument         |
+    | 123550                  | datum einde geldigheid Nederlands reisdocument          |
+    | 123560                  | datum inhouding/vermissing Nederlands reisdocument      |
+    | 123570                  | aanduiding inhouding/vermissing Nederlands reisdocument |
+
+  Abstract Scenario: '<type>' is in onderzoek en de houder velden wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens 
+    | naam                            | waarde                            |
+    | aanduiding in onderzoek (83.10) | <aanduiding persoon in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20230201                          |
+    En de persoon heeft een 'reisdocument' met de volgende gegevens
+    | naam                                        | waarde    |
+    | soort reisdocument (35.10)                  | PN        |
+    | nummer reisdocument (35.20)                 | NE3663258 |
+    | datum einde geldigheid reisdocument (35.50) | 20240506  |
+    Als gba reisdocumenten wordt gezocht met de volgende parameters
+    | naam               | waarde                         |
+    | type               | RaadpleegMetReisdocumentnummer |
+    | reisdocumentnummer | NE3663258                      |
+    | fields             | houder                         |
```

Comment:
hoeft niet nu, maar mooi zou zijn dat er ook een variant is met fields=houder.burgerservicenummer